src/pages/library/dataBrowser-utils.js

```diff
@@ -80,7 +80,7 @@ const normalizeDataset = dataset => {
     dataReleasePolicy,
     contacts, curators, contributorNames,
     dataType, dataModality,
-    access: _.intersection(dataset.roles, ['reader', 'owner']).length > 0 ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
+    access: _.intersection(_.map(_.toLower, dataset.roles), ['reader', 'writer', 'owner']).length > 0 ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
```

Comment:
I don't think we should use `toLower` here; the UI and the service are both things we control so we should be able to have both ends agree on an API where the client knows what format the server is returning.

Regarding `writer`, I've covered this in the other PR but I don't think we should add it until we have a use case for it in our API/UI.

Reply:
For `toLower`, I agree, but wanted to have it be backwards compatible. Currently it returns lowercase. Rawls returns uppercase, and TDR returns lowercase. Maybe the solution is to just define our enum lowercase, and not have the toLower here? That would be simpler, but I don't know what the conventions are around enum casing in all of Terra. My understanding was uppercase was the approach, since I have seen uppercase values in both Sam and Rawls, and was surprised we were using lowercase in TDR.